make/photon/prepare/commands/gencerts.py

```diff
@@ -0,0 +1,26 @@
+import os
+import click
+import pathlib
+from subprocess import check_call, PIPE, STDOUT
+
+from utils.cert import openssl_installed
+from utils.misc import get_realpath
+
+gen_tls_script = pathlib.Path(__file__).parent.parent.joinpath('scripts/gencert.sh').absolute()
+
+@click.command()
+@click.argument('path')
+def gencert(path):
```

Comment:
This is a little trivial but can we provide help to each sub-command?

Reply:
help will automatically generated by `click`